app/EatUp_Client/app/src/main/java/com/github/eatup_client/api/ProductApiService.java

```diff
@@ -0,0 +1,317 @@
+package com.github.eatup_client.api;
+
+import android.content.Context;
+import android.util.Log;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+
+import com.github.eatup_client.model.Allergy;
+import com.github.eatup_client.model.Category;
+import com.github.eatup_client.model.OrderProduct;
+import com.github.eatup_client.model.Product;
+import com.github.eatup_client.model.MVCManager;
+import com.github.eatup_client.model.Session;
+import com.github.eatup_client.model.SessionId;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+
+import java.io.IOException;
+import java.util.List;
+
+import okhttp3.Cache;
+import okhttp3.OkHttpClient;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+import retrofit2.http.Body;
+import retrofit2.http.GET;
+import retrofit2.http.Headers;
+import retrofit2.http.POST;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+/**
+ * Service class for making API calls related to products.
+ */
+public class ProductApiService {
+
+    private static final String BASE_URL = "http://159.69.216.101/api/v1/";
```

Comment:
This constant will allow an easy setup for dynamic urls in the future. Great work!